homeassistant/components/tuya/switch.py

```diff
@@ -290,6 +290,15 @@
             entity_category=ENTITY_CATEGORY_CONFIG,
         ),
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Switch",
+            icon="mdi:sprinkler-variant",
+            entity_category=ENTITY_CATEGORY_CONFIG,
```

Comment:
This is the main switch to turn in on/off right? Shouldn't that be a primary entity?